packages/jellyfish-api-core/__tests__/category/token/burnTokens.test.ts

```diff
@@ -1,16 +1,18 @@
 import { TestingGroup } from '@defichain/jellyfish-testing'
+import { BigNumber } from '../../../src'
 
 describe('burnTokens', () => {
-  const tGroup = TestingGroup.create(2)
+  const tGroup = TestingGroup.create(3)
   const symbolDBTC = 'DBTC'
-  let account0: string, account1: string
+  let account0: string, account1: string, account2: string
   let idBTC: string
 
   beforeEach(async () => {
     await tGroup.start()
 
     account0 = await tGroup.get(0).generateAddress()
     account1 = await tGroup.get(1).generateAddress()
+    account2 = await tGroup.get(1).generateAddress()
```

Comment:
Aren't these the same? @shohamc1

Reply:
Doesn't affect the test, but fixed nonetheless: [f2c2dd1](https://github.com/JellyfishSDK/jellyfish/pull/1860/commits/f2c2dd1c3810a719a0e7a4c96481bd0eb9afe2c9)